storage/statedb/cache_redis_test.go

```diff
@@ -113,9 +112,8 @@ func _TestNewRedisCache_Set_LargeData(t *testing.T) {
 	assert.Equal(t, bytes.Compare(value, retValue), 0)
 }
 
-// testNewRedisCache_Timeout test timout feature of redis client.
-// INFO: Enable it just when you want to test.
-func testNewRedisCache_Timeout(t *testing.T) {
+// TestNewRedisCache_Timeout test timout feature of redis client.
+func _TestNewRedisCache_Timeout(t *testing.T) {
```

Comment:
Can't we just add `- image: circleci/redis:[TAG]` to enable redis in Klaytn repo? @whoisxx

Reply:
Probably... I will add it in the next PR.